lib/manager/fvm/extract.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { loadFixture } from '../../../test/util';
+import { extractPackageFile } from './extract';
```

Comment:
```suggestion
import { extractPackageFile } from '.';
```
Better code coverage. 😉